std/algorithm/sorting.d

```diff
@@ -2274,9 +2294,9 @@ private template TimSortImpl(alias pred, R)
     alias T = ElementType!R;
 
     alias less = binaryFun!pred;
-    alias greater = (a, b) => less(b, a);
-    alias greaterEqual = (a, b) => !less(a, b);
-    alias lessEqual = (a, b) => !less(b, a);
+    bool greater()(auto ref T a, auto ref T b) { return less(b, a); }
+    bool greaterEqual()(auto ref T a, auto ref T b) { return !less(a, b); }
+    bool lessEqual()(auto ref T a, auto ref T b) { return !less(b, a); }
```

Comment:
Not introduced here, but it seems these are superfluous (everything can be elegantly expressed with `less`). `greater` is not even used.

Reply:
I would agree for `greaterEqual(a, b)` vs `!less(a,b)`. However, not for `lessEqual(a, b)` vs `!less(b, a)`. More state for brain to track.